src/java.desktop/windows/classes/sun/awt/windows/WScrollPanePeer.java

```diff
@@ -162,7 +162,7 @@ private void postScrollEvent(int orient, int type,
      * operation.
      */
     @SuppressWarnings("serial") // JDK-implementation class
-    class ScrollEvent extends PeerEvent {
+     static class ScrollEvent extends PeerEvent {
```

Comment:
There's an extra space before `static` modifier. Is it intentional?

Reply:
fixed